pkg/webhooks/hiveextension/v1beta1/agentclusterinstall_validation_hook_test.go

```diff
@@ -179,6 +202,29 @@ var _ = Describe("ACI web validate", func() {
 			operation:       admissionv1.Update,
 			expectedAllowed: false,
 		},
+		{
+			name: "Test AgentClusterInstall.Spec update allowed for provision fields when Install finished",
```

Comment:
```suggestion
			name: "Test AgentClusterInstall.Spec update allowed for provision fields when Install failed",
```

Reply:
i don't see why we should allow editing for installation failure, it should not change anything just wonder if there is a specific reason to allow it

Reply:
I am not sure how their CI/CD is operating. I thought to enable the change on all complete scenarios. WDYT?

Reply:
it's a critical issue, i'm not sure how the system will react in this case, after all they cannot reset the installation so it will just stay in error, they will need to re-create the clsuter